user/serializers.py

```diff
@@ -22,3 +30,35 @@ def update(self, instance, validated_data):
             user.save()
 
         return user
+
+
+class AuthTokenSerializer(serializers.Serializer):
+    email = serializers.CharField(label=_("Email"), write_only=True)
+    password = serializers.CharField(
+        label=_("Password"),
+        style={"input_type": "password"},
+        write_only=True,
+        trim_whitespace=False,
+    )
+    token = serializers.CharField(label=_("Token"), read_only=True)
```

Comment:
According to the checklist, you should not use `serializers.CharField` for fields that are meant to handle images. Ensure that the `email`, `password`, and `token` fields are correctly typed, as they are not image fields. If any of these fields are intended for images, consider using `serializers.ImageField` instead.